sal_interfaces/TunableLaser/TunableLaser_Events.xml

```diff
@@ -35,4 +35,37 @@
       <Count>1</Count>
     </item>
   </SALEvent>
+  <SALEvent>
+    <Subsystem>TunableLaser</Subsystem>
+    <EFDB_Topic>TunableLaser_logevent_burstModeSet</EFDB_Topic>
+    <item>
+      <EFDB_Name>timestamp</EFDB_Name>
+      <Description>Time in seconds of the event.</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>second</Units>
+      <Count>1</Count>
+    </item>
+  </SALEvent>
+  <SALEvent>
+    <Subsystem>TunableLaser</Subsystem>
+    <EFDB_Topic>TunableLaser_logevent_continuousModeSet</EFDB_Topic>
+    <item>
+      <EFDB_Name>timestamp</EFDB_Name>
+      <Description>Time in seconds of the event.</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>second</Units>
+      <Count>1</Count>
+    </item>
```

Comment:
Is this timestamp really needed? Would the value in private_sndStamp be good enough? If so, please delete it.